website/data/docs-nav-data.json

```diff
@@ -1064,7 +1089,7 @@
           },
           {
             "title": "OIDC Providers",
-            "path": "auth/jwt_oidc_providers"
+            "path": "auth/jwt/oidc_providers"
```

Comment:
This was a nesting issue that should have been caught previously, but wasn't due to an issue within `docs-page` which has since been resolved. I've added a corresponding `redirect` to ensure the old URL still works.